content/news/buzz/2017-google-summer-of-code-final-submission-adam-boniecki/contents.lr

```diff
@@ -0,0 +1,54 @@
+title: 2017 Google Summer of Code - Final Submission
+---
+author: Adam Boniecki
+---
+body:
+
+With Google Summer of Code 2017 program nearing its end, it is time to summarize what I got done during the summer working on `Batavia </project/projects/bridges/batavia/>`__.
+
+Batavia is a part of Pybee's BeeWare collection of projects. I posted my proposal in `this GitHub thread <https://github.com/pybee/batavia/issues/483>`__ and kept it updated with my progress on a weekly basis.
+
+Note that by the end of GSoC, we have decided to diverge from the proposal and forgo implementation of `contextlib` in favor of support for Python 3.6 2-byte wide opcodes.
+
+Overall it was great learning experience and fun. Big thanks to my mentors Russel Keith-Magee and Katie McLaughlin, and the whole Pybee community.
```

Comment:
If you had any take-aways, or surprising things learned while working on the project, I'd be super curious to hear about them and I'm sure others would as well. This might be a good place to add them, either before or after the thank-yous.